wpeverest/everest-forms
# Tweak - Removed unnecessary CSS to fix Multi-part button alignment

### All Submissions:






### Changes proposed in this Pull Request:



This PR removed unnecessary CSS to fix Multi-part button alignment issue.
### How to test the changes in this Pull Request:

1. Install the Multi-part addon.
2. Create a form and enable multi-part form from form setting.
3. Check all button alignment option on the front-end.

### Types of changes:




### Other information:




### Changelog entry

> Tweak - Removed unnecessary CSS to fix Multi-part button alignment
